R/SpatialExperiment.R

```diff
@@ -211,7 +211,7 @@ SpatialExperiment <- function(...,
     imgData=NULL) {
     
     sce <- SingleCellExperiment(...)
-    spe <- .sce_to_spe(sce=sce,
+    spe <- .sce_to_spe(sce=sce, ## what happened to the image_id?
```

Comment:
I think we can remove this comment (line 214) and add `image_id=image_id` between lines 221 and 222.

Reply:
done

Reply:
remove the comment?

Reply:
done